napari/_qt/qt_main_window.py

```diff
@@ -1019,6 +1019,21 @@ def resize(self, width, height):
         """
         self._qt_window.resize(width, height)
 
+    def setGeometry(self, left, top, width, height):
```

Comment:
```suggestion
    def set_geometry(self, left, top, width, height):
```

Camel case in napari codebase is reserved for Qt-based objects.